vignettes/visualization-example.Rmd

```diff
@@ -0,0 +1,155 @@
+---
+title: "Example of interacting with and visualizing EFI standard data"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Data Access and Visualization}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, message=FALSE}
+library(EML)
+library(emld)
+library(lubridate)
+library(tibble)
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+```
+
+
+Visualizing the simple, example forecast of population growth of two interacting species
+
+========================================================
+
+## Parsing metadata 
+
+
+First we load the data using `EML`.
+
+```{r}
+eml_filename <- system.file("extdata", "forecast-eml.xml", package="EFIstandards")
+md <- read_eml(eml_filename)  #you could replace the variable here with a filename form a downloaded data package from a repository
+```
+
+Then we can examine the details. First, the title and abstract and who made it.
+
+```{r basic}
+eml_get(md, "title")
+eml_get(md, "abstract")
+eml_get(md, "creator")
+```
+
+Then diving in, the coverage and keywords
+
+```{r details}
+eml_get(md, "coverage")
+eml_get(md, "keywordSet")
+```
+
+Looks like it's a population level timeseries forecast of `r eml_get(md, "geographicDescription")[[1]]` for two species:
+
+```{r species}
+# these seem a bit of a pain to pull out
+spps <- eml_get(md, "taxonRankValue")
+sp1 <- paste(spps[1:2], collapse=" ")
+sp2 <- paste(spps[3:4], collapse=" ")
+sp1
+sp2
+```
+
+## Reading structured methods
+
+Load the additional metadata (link to  defs of additional metadata or paste here?)
```

Comment:
I would just say "see documentation" rather than linking to the documentation because the doc link changes frequently (currently in gDocs rather than Rmd files within the package itself -- the latter would be better but I don't have the time/bandwidth to translate and gdoc has been easier for group discussions)